.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.18.7
```

Comment:
remove and add to .gitignore, I think this is an artifact of asdf

Reply:
yeah, `asdf` uses it to track which version of golang we're using to run/build this. I can remove it, but it helps to keep people in sync beyond just "use go 1.18 ... or don't , go doesn't complain too loudly" =P I can remove it though.